ktxspec.adoc

```diff
@@ -849,8 +854,9 @@ value must be 0.
 An array, `levels`, giving the offset from the start of the file and
 compressed and uncompressed byte sizes of the image data for each
 mip level within the <<_mip_level_array,_Mip Level Array_>> The array is ordered
-starting with stem:[level_{base}] (the level with the largest size images)
-at index _0_. Image for stem:[level_p] will be found at index _p_.
+starting with stem:[\textit{level}_{base}] (the level with the largest size images)
```

Comment:
`base` needs wrapping too.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/227/commits/f37f20c3bdd5a13b8b6dedb75efb29ebfceaafd3.